lib/config-var.js

```diff
@@ -45,6 +45,7 @@ async function update(pg_pool, app_uuid, app_name, space_name, space_tags, org,
   }
   let existing_config_vars = await common.alamo.config.set.request(pg_pool, app_name, space_name)
   let service_data = await addons.service_config_vars(pg_pool, app_uuid, space_name, app_name)
+  let reserved_config_vars = ["PORT", "AKKERIS_DEPLOYMENT", "AKKERIS_APPLICATION", "AKKERIS_SPACE", "AKKERIS_GIT_SHA1", "AKKERIS_GIT_BRANCH", "AKKERIS_GIT_REPO"]
   let config_out = from_alamo_to_config_var(existing_config_vars);
```

Comment:
While working on the PORT change I decided to make reserved env vars impossible to accidently set, the CLI and UI intervene when they see PORT as a config var and remove it (but use the formation change to update the port). If someone is going directly against the API this will prevent the PORT variable from being set.